lib/src/display/stage.dart

```diff
@@ -253,6 +255,12 @@ class Stage extends DisplayObjectContainer {
     _updateCanvasSize();
   }
 
+  /// Gets and sets the qualification of the Stage for rendering.
```

Comment:
This would be the only change necessary to provide StageXL users with the possibility to implement their own idle mode logic.